go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/BurntSushi/toml v1.2.1
 	github.com/ClickHouse/clickhouse-go v1.5.4
 	github.com/DATA-DOG/go-sqlmock v1.5.0
+	github.com/INSRapperswil/cisco-xr-telemetry-proto v1.0.1
```

Comment:
Can you please try to file a PR against https://github.com/cisco-ie/nx-telemetry-proto to add your generated code!? It would be nice to not have to randomly switch repositories for this rather critical part...

Reply:
The mentioned repo is for Cisco NXOS and not IOS-XR, so I think a PR would be declined anyway.
Unfortunately, the only +/- current proto file including the delete field is for [IOS-XR](https://github.com/ios-xr/model-driven-telemetry) (working also for NXOS). Cisco doesn't provide the Go implementation for this proto file -> this is why we have created this repo. An advantage is that we can at least react to updated proto files fast.

Another option would be to save the proto files directly in the telegraf repo, but I think this was not intended to do.

Reply:
Update: requested cisco to create its own repo - waiting for an answer.

Reply:
@srebhan @powersj FYI: Finally, I got the clearance to do a PR on the nx-telemetry-proto repo with the latest files. I will do it tomorrow and change the import path in this PR.